src/renderer/component/fileList/view.jsx

```diff
@@ -70,8 +71,9 @@ class FileList extends React.PureComponent<Props, State> {
   getChannelSignature = (fileInfo: FileInfo) => {
     if (fileInfo.value) {
       return fileInfo.value.publisherSignature.certificateId;
+    } else if (fileInfo.metadata) {
```

Comment:
What's going on here that this can have two different data structures?

If the daemon is inconsistent with it's responses, I'd like to get issues opened to clean this up.